matplotlib version issue (#1002)
CCI-Tools/cate: CHANGES.md

  - Parameter `crs` of function `cate.ops.io.read_geo_data_frame` 
* Fixed bug with user preferences not being saved correctly.
  ([#146](https://github.com/CCI-Tools/cate-app/issues/146))

> * Fixed environment not building with matplotlib version `<3.3.0` ([#929](https://github.com/CCI-Tools/cate/issues/929))

Reviewer: ```suggestion
* Fixed environment not building with matplotlib version `<3.3.0`. (#929)
```

> * Fixed the operator `animate`not working with the current xarray version (0.18)

Reviewer: ```suggestion
* Fixed the operation `animate_map` that stopped working with xarray version 0.18.0.
```
